openvm/src/lib.rs

```diff
@@ -486,19 +487,20 @@ pub enum ExtendedVmConfigPeriphery<F: PrimeField32> {
 
 #[derive(Clone, Serialize, Deserialize, Default, Debug, Eq, PartialEq)]
 pub struct AirMetrics {
+    /// The main AIR statistics (i.e., number of main columns, constraints, and bus interactions)
+    pub air_stats: AirStats,
```

Comment:
why does it have stats and widths?

Reply:
See the comment below. The widths are OpenVM-specific, the `AirStats` are not. I can also undo this change and compute the air stats from the air metrics...